README.md

```diff
@@ -66,16 +66,16 @@ details, see the manual.
 
 ### Initialization scripts
 
-When yash is started as a login shell, it reads `~/.yash_profile`. This
+When yash is started as a login shell, it reads `~/.config/yash/yash_profile`. This
 file is a shell script in which you define environment variables using
 the export command.
 
-When yash is started for an interactive use, it reads `~/.yashrc` (after
-reading `~/.yash_profile` if it is a login shell also). In this file,
-you make other configurations such as aliases, prompt strings, key
-bindings for command line editing, and command-not-found handler.
-Use the [share/initialization/sample](share/initialization/sample) file
-as a template for your `~/.yashrc`.
+When yash is started for an interactive use, it reads `~/.config/yash/yashrc`
+or `~/.yashrc` (after reading `~/.config/yash/yash_profile` if it is a login
+shell also). In this file, you make other configurations such as aliases,
+prompt strings, key bindings for command line editing, and command-not-found
+handler. Use the [share/initialization/sample](share/initialization/sample)
+file as a template for your `~/.config/yash/yashrc`.
```

Comment:
> users with non-default XDG_CONFIG_HOME can safely place their rc file anywhere they want instead of the default ~/.config/yash/yashrc.

I will update the conditionals to actually check XDG_CONFIG_HOME instead of using `~/.config/yash` as a hard coded path.

> Also, ~/.config/yash/yashrc should be ~/.config/yash/rc.


I chose to use `~/.config/yash/yashrc` for consistency, however, I think removing `yash` in the filename would be cleaner.